harmony_changelog.md

```diff
@@ -2,30 +2,38 @@
 
 ## Unreleased
 
+### Fixed
+* Increase clock font in Relaxation, Meditation, Power nap mode
+
+### Added
+* Added gradual alarm volume increase
+* Added progress bar for all volume control windows
+
+### Changed / Improved
+* Optimize the way Relaxation is loading music files
+* Factory reset removes user files
+* Changed countdown progress bar design in Relaxation, Meditation and Power Nap apps
+* Removed minus sign in progress countdown timers in Relaxation, Meditation and Power Nap apps
+* Extended volume scale from 10 to 15 point scale
+* Modified volume approximation function for better user experience
```

Comment:
```suggestion
* Modified volume control characteristic for better user experience
```

I know that you're not the author of this entry, but it's somewhat imprecisely formulated - we're not using any "volume approximation", we've just changed the volume knob characteristics, i.e. the way the quantifiable volume level translates into perceived loudness.